modules/twitch-notify/events/botReady.js

```diff
@@ -0,0 +1,52 @@
+const {confDir} = require('../../../main');
+const config = require(`${confDir}/twitch-notify/config.json`);
+
+const { ApiClient } = require('twitch');
+const { ClientCredentialsAuthProvider } = require('twitch-auth');
+
+const ClientID = config['Twitch-ClientID'];
+const ClientSecret = config['ClientSecret'];
+const authProvider = new ClientCredentialsAuthProvider(ClientID, ClientSecret);
+const apiClient = new ApiClient({ authProvider });
+
+let live = false;
+
+function twitch_notify(client) {
+  function replacer(msg, username, game) {
+    msg = msg.split('%streamer%').join(username)
+        .split('%game%').join(game)
+        .split('%url%').join('https://twitch.tv/' + username.toLowerCase());
```

Comment:
Please use `embedType(msg, {'%game%': game, '%url': 'https://twitch.tv/' + username.toLowerCase() }` from /src/functions/helpers to allow users to use embeds. Please consider removing the `twicht_notify` function in that case completely as it is not longer really needed.